.changeset/fast-walls-carry.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-plugin-api': patch
+---
+
+Introduced an optional metadata for the components, which can contain a configurable data.
+The author of the component might define the configurable places of the component, and the user of this
```

Comment:
This needs an update to reflect the new changes. Probably worth splitting the minor TS changes into separate changesets too

Reply:
what kind of minor TS changes you have in mind?